MonoGame.Framework/Platform/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -378,7 +358,7 @@ internal void CreateSizeDependentResources()
                 };
```

Comment:
You can move setting Flags in the SwapChainDescription1 initializer here.
Also don't need the WINDOWS_UAP check since it's set to none for WindowsDX which is fine.

```suggestion
                    Flags = swapChainFlags,
                };
```